habitat/datasets/rearrange/rearrange_generator.py

```diff
@@ -421,15 +458,63 @@ def generate_single_episode(self) -> Optional[RearrangeEpisode]:
         }
 
         ep_scene_handle = self.generate_scene()
+        self.navmesh_settings = NavMeshSettings()
+        self.navmesh_settings.set_defaults()
+        self.navmesh_settings.agent_radius = 0.3
+        self.navmesh_settings.agent_height = 1.5
+        self.navmesh_settings.agent_max_climb = 0.05
```

Comment:
I think these should live in a config, or maybe adding a comment about why these are manually modified here.
```suggestion
        # In order to make sure the robot will be able to navigate properly, 
        # we increase the size of the agent here.
        self.navmesh_settings.agent_radius = 0.3
        self.navmesh_settings.agent_height = 1.5
        self.navmesh_settings.agent_max_climb = 0.05
```

Reply:
Code was anyways redundant. I replaced it with loading in the pre-computed navmesh for that scene.